src/components/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -26,22 +39,94 @@ export interface Props {
  * import {BreadcrumbsItem} from "@chanzuckerberg/eds";
  * ```
  *
- * A single breadcrumb, to be used in the Breadcrumbs component.
+ * A single breadcrumb subcomponent, to be used in the Breadcrumbs component.
  */
-export const BreadcrumbsItem = ({ className, text, href, ...other }: Props) => {
-  const componentClassName = clsx(styles['breadcrumbs__item'], className, {});
+export const BreadcrumbsItem = ({
+  dropdownMenuItems,
+  className,
+  href,
+  text,
+  variant,
+  ...other
+}: Props) => {
+  const [isActive, setIsActive] = React.useState(false);
+
+  /**
+   * Closes the dropdown menu if there is focus or click outside.
+   */
+  const handleBlur = (event: React.FocusEvent<HTMLElement, Element>) => {
+    if (!event.currentTarget.contains(event.relatedTarget) && isActive) {
+      setIsActive(false);
+    }
+  };
+
+  const componentClassName = clsx(
+    styles['breadcrumbs__item'],
+    variant === 'back' && styles['breadcrumbs__item-back'],
+    className,
+  );
+
+  const ellipsisButtonClassName = clsx(
+    styles['breadcrumbs__link'],
+    styles['breadcrumbs__ellipsis'],
+  );
+
+  const dropdownMenuClassName = clsx(
+    styles['breadcrumbs__dropdown-menu'],
+    isActive && styles['breadcrumbs__dropdown-menu--active'],
+  );
+
+  const getInteractionElement = () => {
+    if (variant === 'collapsed') {
+      /* The collapsed variant is a button with ellipsis. Interaction spawns a dropdown containing the collapsed breadcrumb links. */
+      return (
+        <>
+          <button
+            className={ellipsisButtonClassName}
+            onClick={() => {
+              setIsActive(!isActive);
+            }}
+          >
+            ...
+          </button>
+          <DropdownMenu
+            className={dropdownMenuClassName}
+            isActive={isActive}
+            onBlur={handleBlur}
+          >
+            {dropdownMenuItems}
+          </DropdownMenu>
+        </>
+      );
+    } else if (variant === 'back') {
+      /* The back variant is a left pointing icon that usually links to the second last breadcrumb href. */
+      return (
+        <a className={styles['breadcrumbs__link']} href={href as string}>
+          <Icon
+            className={styles['breadcrumbs__back-icon']}
+            name="chevron-left"
+            purpose="informative"
+            title={href as string}
```

Comment:
Could we use `text` here?

Reply:
Yes, that would make more sense, thanks!